pkgs/tools/nix/info/info.sh

```diff
@@ -69,8 +69,14 @@ nixev() {
     nix-instantiate --eval --strict -E "$1"
 }
 
+# use `nix eval` command
+nixev2() {
+    nix eval --raw "$1"
```

Comment:
The nix3 cli is only available in nix 2.4+, and is guarded by experimental flags. There should be a version check and the relavent flags should be set.

Reply:
Change has made according to @figsoda 's comment. There is no need to use experimental command now.